stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/DeliveryStatusDecorator.kt

```diff
@@ -88,5 +89,9 @@ internal class DeliveryStatusDecorator : BaseDecorator() {
                 hideIndicator()
             }
         }.exhaustive
+
+        if (data.message.isEphemeral()){
```

Comment:
Hiding indicator for ephemeral messages

Reply:
We should do these amongst the checks in the beginning of the method